initrd.txt

```diff
@@ -0,0 +1,1397 @@
+[   12539904]  ./rootfs-0/kernel/x86/microcode/GenuineIntel.bin
+[    5343472]  ./rootfs-1/usr/lib64/libcrypto.so.3*
+[    3208664]  ./rootfs-1/usr/lib/systemd/libsystemd-shared-252.so*
+[    2591624]  ./rootfs-1/usr/lib/gcc/x86_64-cros-linux-gnu/13/libstdc++.so.6.0.32*
+[    2058480]  ./rootfs-1/usr/lib/systemd/libsystemd-core-252.so*
+[    1933576]  ./rootfs-1/usr/lib64/libc.so.6*
+[    1845305]  ./rootfs-1/usr/lib/firmware/phanfw.bin
+[    1649076]  ./rootfs-1/usr/lib/firmware/mellanox/mlxsw_spectrum-13.2010.1006.mfa2
```

Comment:
Spectrum is a network switch, i think we could get rid of these firmware files.

Reply:
` 51M Mar  6 11:52 flatcar_production_image.vmlinuz` with CONFIG_MLXSW_SPECTRUM and CONFIG_MLXSW_SPECTRUM_DCB removed vs 55M the HEAD one.

Reply:
```/dev/vda1          129039      59962      69078  47% /boot``` with the removal on AMD64.

Reply:
Created PR https://github.com/flatcar/scripts/pull/1734 as a first version of the effort to decrease the initrd size.